src/config/groups/members.cpp

```diff
@@ -162,6 +162,17 @@ size_t Members::size() const {
     return 0;
 }
 
+bool Members::has_pending_send(std::string pubkey_hex) const {
+    return pending_send_ids.count(pubkey_hex);
+}
+
+bool Members::set_pending_send(std::string pubkey_hex, bool pending) {
+    if (pending)
+        return pending_send_ids.insert(pubkey_hex).second;
+    else
+        return pending_send_ids.erase(pubkey_hex);
+}
```

Comment:
Should we put the `_needs_dump = true;` setting in here (if the `insert`/`erase` actually do something)?

Just thinking that if someone happens to call `set_pending_send` somewhere in the future that doesn't also mutate the config then it would be easy to forget to add `_needs_dump = true;` call and result in the dump not being saved